.travis.yml

```diff
@@ -114,12 +115,13 @@ before_install:
   - curl -fsSkL https://raw.githubusercontent.com/openmeeg/ci-utils/master/travis/travis_tools.sh > x.sh && source ./x.sh
 
   ############################################################################
-  # Install MKL (if necessary)
+  # Install MKL if necessary
   ############################################################################
 
   - |
     if [[ "${BACKEND}" == *"Intel"* ]]; then
-      curl -fsSkL https://raw.githubusercontent.com/openmeeg/ci-utils/master/travis/install_mkl.sh > install_mkl.sh && source ./install_mkl.sh
+      curl -fsSkL https://raw.githubusercontent.com/openmeeg/ci-utils/master/travis/install-mkl.sh > install_mkl.sh && source ./install_mkl.sh
+      echo "[[${MKLROOT} -- ${MKL_ROOT}]]"
```

Comment:
why do you need to change anything to MKL build system?

Reply:
I discovered that they now provide apt repos.... This is probably faster and since it seemed there was problems with getting mkl, I tried this. It is simpler and should also be faster....